database/gormdb_assignment.go

```diff
@@ -69,7 +68,9 @@ func (db *GormDB) GetAssignmentsByCourse(courseID uint64, withGrading bool) ([]*
 	if withGrading {
 		for _, a := range assignments {
 			var benchmarks []*pb.GradingBenchmark
-			if err := db.conn.Where("assignment_id = ?", a.ID).Find(&benchmarks).Error; err != nil {
+			if err := db.conn.Where("assignment_id = ?", a.ID).
```

Comment:
Suggest to move `.Where()` to a separate line to make it line up with the next `Where()` clause.